src/eventBus.ts

```diff
@@ -172,23 +171,21 @@ class EventBus implements Disposable {
      *
      * @param name - The name of the event.
      * @param handler - The handler function for the event.
-     * @param thisArg - The `this` context used when invoking the handler function.
```

Comment:
Why remove this?

Reply:
This was to satisfy `no-unbound-method`. ESLint isn't smart enough to dive into this implementation to realize we are binding `this`. In practice, when setting up a listener, we never bound anything other than the `this` that already existed in lexical scope, so I preferred to use arrow functions, which capture `this`, rather than this binding.

Reply:
In general, this topic is also highly relevant to personal habits. I don't engage in too much discussion. The `thisArg` parameter is designed similarly in many APIs within VS Code as well. Regarding this rule, we can consider it as a suggestion rather than a limitation.

Reply:
You're right, it's a style thing. We could follow the way VSCode APIs are set up, but we wouldn't be able to reap the benefits of this rule, which eliminate an entire class of bug. 

Would you be amenable to replacing the arrow functions with `this.foo.bind(this)`? It would have the same effect, and satisfy the linter.